apps/astarte_appengine_api/test/astarte_appengine_api/v2_device_test.exs

```diff
@@ -0,0 +1,51 @@
+defmodule Astarte.AppEngine.API.V2DeviceTest do
+  use ExUnit.Case, async: true
+  use ExUnitProperties
+  use Astarte.Test.Cases.Device
```

Comment:
This `use` hides the setup step(s) needed to populate the db, so it is unclear where e.g. devices selected at line 39 do come from.  It should be possible to clearly identify what is being created, what operation is being performed and what the expected output would be by just looking at a single test.
I find that explicitly implementing `setup_all` and `setup` in the test case helps doing so.

The same goes for the other tests on groups and interfaces.